camel/src/main/java/com/github/theprez/manzan/configuration/DestinationConfig.java

```diff
@@ -65,15 +67,21 @@ public synchronized Map<String, ManzanRoute> getRoutes(CamelContext context) {
                     break;
                 case "kafka":
                     final String topic = getRequiredString(name, "topic");
-                    ret.put(name, new KafkaDestination(name, topic, format, getUriAndHeaderParameters(name, sectionObj, "topic")));
+                    ret.put(name, new KafkaDestination(context, name, topic, format, componentOptions, getUriAndHeaderParameters(name, sectionObj, "topic")));
+                    break;
+                case "activemq":
+                    final String destName = getRequiredString(name, "destinationName");
+                    String destType = getOptionalString(name, "destinationType");
+                    destType = destType.equals("topic") ? "topic" : "queue";
```

Comment:
If `destinationType` is not set as it is optional, `.equals` would cause a null pointer exception?